src/react-native.js

```diff
@@ -82,7 +82,7 @@ const ReactNative = {
   StyleSheet: require('./api/StyleSheet'),
   TimePickerAndroid: require('./api/TimePickerAndroid'),
   UIManager: require('./NativeModules/UIManager'),
-  VibrationIOS: require('./api/VibrationIOS'),
+  Vibration: require('./api/Vibration'),
```

Comment:
Hmm yes that does sound like a better idea! Intended for this to be a  large PR initially so did a lot of changes. Will revert this